controller/node_controller.go

```diff
@@ -309,7 +363,20 @@ func (nc *NodeController) syncNode(key string) (err error) {
 				}
 			}
 			node.Status.Conditions[types.NodeConditionTypeReady] = condition
+			break
+		}
+	}
+	// check physical node down
+	if removedNode {
+		condition := types.GetNodeConditionFromStatus(node.Status, types.NodeConditionTypeReady)
+		if condition.Status != types.ConditionStatusFalse {
+			condition.LastTransitionTime = util.Now()
+			nc.eventRecorder.Eventf(node, v1.EventTypeWarning, types.NodeConditionReasonPhysicalNodeDown, "Physical node missing: Node %v has removed from cluster without manager pod running on it", node.Name)
```

Comment:
Use the same string for this one and below.

Also:

`Kubernetes node missing: node %v has been removed from the cluster and there is no manager pod running on it`